examples/cis5-smart-contract-wallet/src/lib.rs

```diff
@@ -75,27 +76,27 @@ pub enum Event {
     /// The event tracks every time a CCD amount received by the contract is
     /// assigned to a public key.
     #[concordium(tag = 249)]
-    DepositNativeCurrency(DepositNativeCurrencyEvent),
+    DepositCcd(DepositCcdEvent),
     /// The event tracks every time a token amount received by the contract is
     /// assigned to a public key.
     #[concordium(tag = 248)]
     DepositCis2Tokens(DepositCis2TokensEvent),
     /// The event tracks every time a CCD amount held by a public key is
     /// withdrawn to an address.
     #[concordium(tag = 247)]
-    WithdrawNativeCurrency(WithdrawNativeCurrencyEvent),
+    WithdrawCcd(WithdrawCcdEvent),
     /// The event tracks every time a token amount held by a public key is
     /// withdrawn to an address.
     #[concordium(tag = 246)]
     WithdrawCis2Tokens(WithdrawCis2TokensEvent),
     /// The event tracks every time a CCD amount held by a public key is
     /// transferred to another public key within the contract.
     #[concordium(tag = 245)]
-    InternalNativeCurrencyTransfer(InternalNativeCurrencyTransferEvent),
+    CcdTransfer(CcdTransferEvent),
```

Comment:
To be consistent with the `Deposit and Withdraw` events, I suggest to use `TransferCcd(TransferCcdEvent)`